queries/ocaml/highlights.scm

```diff
@@ -47,11 +47,11 @@
 
 (infix_expression
   left: (value_path (value_name) @function)
-  (infix_operator) @operator
+  operator: (concat_operator) @operator
   (#eq? @operator "@@"))
```

Comment:
Is this ever needed now, considering `(concat_operator)` is already captured with the new changes

Reply:
I'd suggest in the case this isn't necessary anymore (which looks like the case), please use `@_...` capture instead of `@operator`, so the same node isn't captured twice, or remove it entirely